modules/core/src/main/scala/org/tessellation/config/types.scala

```diff
@@ -68,7 +68,12 @@ object types {
   case class RewardsConfig(
     programs: EpochProgress => ProgramsDistributionConfig = mainnetProgramsDistributionConfig,
     rewardsPerEpoch: SortedMap[EpochProgress, Amount] = mainnetRewardsPerEpoch,
-    oneTimeRewards: List[OneTimeReward] = List.empty
+    oneTimeRewards: List[OneTimeReward] = List(
+      // TODO adjust the epoch and balance after we've passed epoch 1352274L
+      // Transferring current balance of 43,320,500 from DAGSTARDUSTCOLLECTIVEHZOIPHXZUBFGNXWJETZVSPAPAHMLXS
+      //  adjusted by 1e8 to build correct transaction amount, one snapshot before 1349352L.
+      OneTimeReward(EpochProgress(1360000L), stardustNewPrimary, TransactionAmount(4_332_050_000_000_000L))
```

Comment:
@sdavidp I see the stardust balance is:
```
4343029488479231
```
not
```
4332050000000000
```

Reply:
Yes, that is why I put the TODO -- the epoch and balance were just placeholders until the morning. They should be correct now.